snippets/prime_number.md

```diff
@@ -0,0 +1,24 @@
+---
+title: prime_number
+tags: prime_number,intermediate
+---
+
+Explain briefly what the snippet does.
+
+- Checks numbers from 2 to (given number - 1), If number is divisible by given numbers.
+- Or, remainder is 0 than its a not a prime number.
+- If number is prime than return 1 and otherwise return 0. 
+
+```py
+def prime_number(number):

```

Comment:
The complexity of this is sub-optimal. Also, I'm pretty sure the `else` part is incorrent. There's a better solution like the second to last snippet [here](https://stackoverflow.com/a/18833870/9298688).